src/Materials/Node/Blocks/Fragment/perturbNormalBlock.ts

```diff
@@ -151,7 +151,7 @@ export class PerturbNormalBlock extends NodeMaterialBlock {
 
         state._emitUniformFromString(this._tangentSpaceParameterName, "vec2");
 
-        let replaceForBumpInfos = this.strength.isConnectedToInputBlock && this.strength.connectInputBlock!.isConstant ? `${state._emitFloat(1.0 / this.strength.connectInputBlock!.value)}` : `1.0 / ${this.strength.associatedVariableName}`;
+        let replaceForBumpInfos = this.strength.isConnectedToInputBlock && this.strength.connectInputBlock!.isConstant ? `${state._emitFloat(this.strength.connectInputBlock!.value)}` : `${this.strength.associatedVariableName}`;

```

Comment:
This was done that way because the shader code expect 1/level:
![image](https://user-images.githubusercontent.com/1306056/111332861-628e2600-862f-11eb-9cc1-e9380edf64e2.png)

Reply:
Ignore me. I read the forum ;)

Reply:
The PBR material is passing the level "as is" whereas the `StandardMaterial` is passing `1/level`: the value is not used at the same place in the shader code.

Reply:
However, we do have a problem because if the block is not used in connection with the PBR block, it will fail with my changes...

Reply:
Oh yeah this is bad..any idea?